README.md

```diff
@@ -1,89 +1,46 @@
-  + I recommend using appium via node module instead of appium.app. This allows the appium server to be started by simply running `appium`
-  + However, feel free to use what you are comfortable with
-+ Xcode 7: For help on installing side-by-side versions of Xcode, see section below
+## Requirements
+
++ Appium server
+    + [Appium 1.6](https://www.npmjs.com/package/appium)
+    + and/or
+    + [Appium desktop app](http://appium.io/downloads.html)
++ Xcode 8
 + IntelliJ
-+ Maven: `brew install maven` if you do not have it installed already
++ [Maven](https://maven.apache.org/index.html) (on macOS you can also easily install it with `brew install maven`)
++ Additional Setup to run on iOS Devices can be found [here](http://appium.io/slate/en/master/?ruby#appium-on-real-ios-devices).
+
+## Setting up the test server
+
+### From command line
 
-###Multiple Xcode Versions###
-If you are trying to run iOS tests, a current limitation is that iOS 10 is not supported. Because of this, Xcode 8 is not supported with the compatible version of Appium. To get around this, you can install Xcode 7 along side Xcode 8.
++ Run `appium-doctor` on command line. Ensure that all checks pass.
++ Run `appium` to start the server
 
-+ Download Xcode 7 from [Apple](http://developer.apple.com/download/more) 
-+ Copy Xcode.app to your desktop (just not /Applications/), and rename to Xcode7.app
-+ Move Xcode7.app to /Applications/.
-+ Open Xcode7 and accept the terms and conditions
+### From Appium desktop app
 
-You're all set to switch between Xcode versions! In order to select which Xcode command line tools are used (Appium needs to use Xcode 7 CLI), you will need to run the command:
-``` bash
-$ sudo xcode-select --switch /Applications/Xcode7.app
-```
-To see which verion of Xcode CLI are currently in use:
-``` bash
-$ xcode-select -p
-```
++ Open app
++ Click start server button
 
-###Running the tests locally###
+## Running the tests locally
 
-####1. Prepare the Appium test server####
-+ Run `appium-doctor` or select the Button in Appium.app. Ensure that all checks pass
-+ Open up Appium.app, select the appropriate platform settings
-+ Set Appium's main settings to:
-    + Server Address: 127.0.0.1:4723
-    + Check For Updates: true
-    + Prelaunch Application: true
-    + Override Existing Sessions: true
-    + New Command Timeout: 7,200s
-+ Go to Appium's iOS settings under advanced and ensure Xcode Path is set to Xcode7.app
-+ Launch the test server
+### From command line
 
-####2a. Running tests from command line####
 + Run `mvn test -P [platform]` from the project directory
     + `[platform]` should be either `android`, `ios`, or `ios-simulator` depending on which platform you want to run your tests on
 
-####2b. Running tests from IntelliJ####
+### From IntelliJ
 + Open the "Maven Projects" tool window
 + Under "Profiles" choose the platfrom you wish to tests on (making sure only one is checked)
```

Comment:
Spelling and grammar errors:
Under "Profiles" choose the *platform* you wish to *test* on (making sure only one is checked)